src/main/resources/resources/logisim/strings/std/std_de.properties

```diff
@@ -796,6 +796,7 @@ TTL7424 = 7424: Vierfach 2-Eingangs-NAND-Gatter (Schmitt-Trigger)
 TTL7427 = 7427: Dreifach 3-Eingangs-NOR-Gatter
 TTL74273 = 74273: Achtfach D-Flipflop mit Clear
 TTL74283 = 74283: 4-Bit-Binär-Volladdierer mit Clear
+# ==> TTL74299 =
```

Comment:
8-Bit-Universalschieberegister mit Tri-State-Ausgängen

Reply:
Done, thanks for the translation. I couldn't find a decent translation online myself.